### PR TITLE
Add dpline CLI

### PR DESCRIPTION
**Pull request by bregman-arie**
Add initial files and structure for CLI support for dpline.

### CODE REVIEW

**Comment by lioramilbaum** [comment]
Do we need to discuss the copyrights?

File: dpline/parser.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import argparse
+
+
+def create_list_parser(subparsers, pparser):
+    """The parser for sub command 'list'."""
+    list_parser = subparsers.add_parser("list", parents=[pparser])
+    list_parser.add_argument('--systems', '-s',
```
Comment (lioramilbaum) [inline dpline/parser.py]: system => subservices
dpline is a service of services

**Comment by lioramilbaum** [comment]
I do not get the usage. please add documentation.

File: dpline/parser.py (hunk 1 above)
Comment (bregman-arie) [reply dpline/parser.py]: > Do we need to discuss the copyrights?

What do you mean?

**Pull request merged**